source/partials/valid_test_account_numbers.erb

```diff
@@ -0,0 +1,16 @@
+<aside class="notice">
```

Comment:
Is it worth making this a warning?

Reply:
> Use class="notice" for blue notes, class="warning" for red warnings, and class="success" for green notes.

Warning is red.
It looks like this:
![image](https://user-images.githubusercontent.com/3105832/82220529-aaacc100-9916-11ea-9986-1de4b5116f7c.png)

Reply:
@deanrecord it looks like something is wrong if it's red.
You think it's a good idea to use red for it?

Reply:
Yeah, it's quite aggressive. Stick with what you had.

Reply:
do we have a yellow or something for 'warning'?

Reply:
Does not look like there is, only blue, red and green.

> Use class="notice" for blue notes, class="warning" for red warnings, and class="success" for green notes.

Reply:
😢 ok